Lombiq.Vsix.Orchard/Options/LogWatcherOptionsPage.cs

```diff
@@ -18,37 +18,46 @@ public class LogWatcherOptionsPage : DialogPage, ILogWatcherSettings
 
         [DisplayName("Enabled")]
         [Category("General Log Watcher Options")]
-        [Description("Enable/disable Log Watcher feature. With this option turned off the Log Watcher won't check if the log file has any new entry.")]
+        [Description("Enable/disable Log Watcher feature. " +
+            "With this option turned off the Log Watcher won't check if the log file has any new entry.")]
         public bool LogWatcherEnabled { get; set; } = true;
 
         [DisplayName("Log file folder paths")]
         [Category("General Log Watcher Options")]
-        [Description("List of relative paths where the log files are located. Values must be relative to the solution file that is currently opened. Wildcards can be used. Use the '|' character to separate values. Example: App_Data/logs|*.Web/App_Data/logs.")]
+        [Description("List of relative paths where the log files are located. " +
+            "Values must be relative to the solution file that is currently opened. " +
+            "Wildcards can be used. Use the '|' character to separate values. Example: App_Data/logs|*.Web/App_Data/logs.")]

```

Comment:
Az ilyen egyszerűen törhető dolgokat mindig max 120 karakternél (a soft limitnél) törjük. A többi itt is.

Reply:
Tehát 120 karakternél, nem 80-nál, és a többi ebben a fájlban lévő stringre is vonatkozik.